eole/predict/inference.py

```diff
@@ -436,7 +436,7 @@ def _process_bucket(bucket_predictions):
             batch_data = self.predict_batch(batch, attn_debug)
 
             predictions = prediction_builder.from_batch(batch_data)
-            is_seq2seq = hasattr(self.model, "encoder") and hasattr(self.model, "decoder")
+            is_seq2seq = self.model.encoder is not None and self.model.decoder is not None
```

Comment:
what is the edge where this is different from previous code?

Reply:
In my test with Eurollm self.encoder was equal to None.